### DISCUSSION TITLE
Model serving

### DISCUSSION THREAD

**Discussion by ghost** [salesforce/TransmogrifAI#536] [Q&A]
Is there a way to serve the models created with TransmogrifAI?

I found PredictionIO useful but it seems that that project was put in the Apache attic.

**Comment by tovbinm** [comment]
We support the serving models using Spark and MLeap. With Spark, simply run `model.score(..)`. For MLeap, have a look here - https://github.com/salesforce/TransmogrifAI/tree/master/local

**Reply by ghost** [reply]
thank you @tovbinm 
I see, so it is possible to export the model in MLeap format and then use MLeap serving to serve it in production.

**Reply by leahmcguire** [reply]
Only stages wrapping spack will be saved as mleap (we have not written converters for stages written in transmogrifai) so it will need to be loaded as a local model in transmogrifai https://github.com/salesforce/TransmogrifAI/tree/master/local

**Comment by leahmcguire** [comment]
Though we don't have any utils for packaging it as an endpoint if that is what you are asking.

**Reply by ghost** [reply]
Yes, the question was a bit ambiguous @leahmcguire 
I was looking for some tools/utils to serve the serialized model in production.
So far I used to use PredictionIO.
